index.html

```diff
@@ -5,77 +5,79 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="styles.css" />
-    <link rel="stylesheet" href="assests/fontAwesome/css/all.css" />
     <title>Personal Portfolio</title>
   </head>
   <body>
     <main class="container">
-      <section class="header-div">
-        <header class="header">
-          <section class="toolbar">
-            <a href="index.html" class="mylogo">GeekyHacks</a>
-            <button class="menu-button" type="button" title="Menu">
-              <i class="fa-solid fa-bars fa-fade fa-2x icon-menu"></i>
+      <header class="header">
+        <h2><a href="index.html" class="mylogo">GeekyHacks</a></h2>
+        <nav class="menu-button">
+          <button type="button" title="Menu">
+            <img src="assests/images/menubutton.svg" alt="menu-button" />
+          </button>
+        </nav>
+      </header>
+
+      <section class="headlines">
+        <h3 class="primary">
+          Hey there. I’m Abdullah. <br />
+          <span class="dev-span"> I’m a software developer</span>
+        </h3>
```

Comment:
- [ ] As requested by the previous code reviewer the main title must be wrapped in the `h1` tag, please 🙏🏽 use the correct notation for the best practices of HTML & CSS practices.